pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -192,14 +193,15 @@ func (t ResolvedPipelineRunTask) isCancelled() bool {
 		if len(t.TaskRuns) == 0 {
```

Comment:
could you please update the docstring of this function to be similar to isFailure?

Reply:
updated the docstring for `isCancelled` - thank you!